platform/security/idp/security-idp-client/src/main/resources/post-binding.html

```diff
@@ -12,27 +11,27 @@
 *
 **/
 -->
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml">
+<!DOCTYPE html>
+<html>
 <head>
-    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
```

Comment:
You don't need the end tags here since it is no longer XHTML.

Reply:
The `charset` attribute isn't needed? The `charset` attribute should be valid HTML 5.